orangecontrib/bio/widgets3/OWdictyExpress.py

```diff
@@ -0,0 +1,235 @@
+from PyQt4.QtCore import Qt
+from Orange.data import Table
+from Orange.widgets.widget import OWWidget
+from Orange.widgets import gui, settings
+from orangecontrib.bio.resolwe import Genapi
+
+from .. import dicty
+
+CallBack = dicty.CallBack
+
+
+Labels = [
+    (" ", " "),
+    ("var.project", "Project"),
+    ("static.name", "Experiment"),
+    ("static.cite", "Citation"),
+    ("var.growth", "Growth"),
+    ("var.treatment", "Treatment"),
+    ("var.strain", "Parental strain")]
+
+
+# Creates line separator
+def h_line():
+    line = QFrame()
+    line.setFrameShape(QFrame.HLine)
+    line.setFrameShadow(QFrame.Sunken)
+    return line
+
+
+class OWdictyExpress(OWWidget):
+
+    name = "dictyExpress"
+    description = "Time-course gene expression data"
+    icon = "../widgets/icons/logo_dE.png"
+    want_main_area = True
+    priority = 37
+
+    inputs = []
+    outputs = [("Data", Table)]
+
+    username = settings.Setting('anonymous@genialis.com')
+    password = settings.Setting('')
+
+    def __init__(self):
+        super().__init__()
+
+        self.res = None
+        self.server = 'https://dictyexpress.research.bcm.edu'
+        self.headerLabels = [x[1] for x in Labels]
+        self.searchString = ""
+        self.items = []
+        self.lastSelected = None  # store last selected customTreeItem
+
+        # Login Section
+
+        box = gui.widgetBox(self.controlArea, 'Login')
+
+        self.namefield = gui.lineEdit(box, self, "username", "Username:", labelWidth=100, orientation='horizontal',
+                                      callback=self.auth_changed)
+
+        self.passfield = gui.lineEdit(box, self, "password", "Password:", labelWidth=100, orientation='horizontal',
+                                      callback=self.auth_changed)
+
+        self.passfield.setEchoMode(QLineEdit.Password)
+
+        self.controlArea.layout().addWidget(h_line())
+
+        self.commit_button = gui.button(self.controlArea, self, "Commit", callback=self.commit)
+        self.handle_commit_button(False)
+
+        self.refresh_button = gui.button(self.controlArea, self, "Refresh", callback=self.refresh)
+        self.handle_cache_button(True)
+
+        gui.rubber(self.controlArea)
+
+        # Experiment Section
+
+        label = QLabel("Available projects:")
+        my_font = QFont()
+        my_font.setBold(True)
+        label.setFont(my_font)
+        self.mainArea.layout().addWidget(label)
+
+        self.mainArea.layout().addWidget(h_line())
+
+        self.filter = gui.lineEdit(self.mainArea, self, "searchString", "Filter:", callbackOnType=True,
+                                   callback=self.search_update)
+
+        self.experimentsWidget = QTreeWidget()
+        self.experimentsWidget.setAlternatingRowColors(True)
+        self.experimentsWidget.setRootIsDecorated(False)
+        self.experimentsWidget.setUniformRowHeights(True)
+        self.experimentsWidget.setSortingEnabled(True)
```

Comment:
You can [set properties of QWidgets in the constructor](http://pyqt.sourceforge.net/Docs/PyQt5/qt_properties.html), e.g.
```py
self.experimentsWidget = QTreeWidget(alternatingRowColors=True,
                                     rootIsDecorated=False,
                                     uniformRowHeights=True,
                                     sortingEnabled=True)